redaxo/src/addons/phpmailer/pages/checkmail.php

```diff
@@ -8,13 +8,10 @@
  */
 
 $addon = rex_addon::get('phpmailer');
-
-$content = $smtpinfo = $mailerDebug = '';
+$content = $smtpinfo = $mailerDebug = $alert = $success = '';
```

Comment:
Da die beiden Variablen unten jeweils nur innerhalb ihres if/else-zweigs verwendet werden, würde ich sie hier nicht initialisieren, sondern als lokale variablen in ihrem jeweiligen zweig betrachten.